my-dsl.md

```diff
@@ -1,39 +1,41 @@
 # Language
 
-_What is the name of the language? Link the name to its webpage
-(if appropriate)._
+Purl
+
+https://arxiv.org/pdf/1606.08708.pdf
+
+(I'm citing from this article using "Purl" and the page number)
 
 # Domain
 
-_Describe the language's domain in five words._
+Modular knitting patterns
 
 # Computational model
 
-_We don't yet have a great definition of the term "computational model".
-For now, try to come up with the clearest, most concise explanation of
-what happens when a program in your DSL runs._
+When Purl runs, it verifies that it is a valid knitting pattern and then creates a knitting
+pattern using standard knitting notation, as well as adding information to help the knitter
+reduce their own mistakes (Purl 7).
 
 # DSL-ness
 
```

Comment:
I totally agree! The limitations of Purl would definitely place it more on the DSL side of the spectrum. Good job describing different functionalities of purl and using those functionalities as evidence for your answer.